src/lib/crypto/kimchi_backend/gadgets/foreign_field.ml

```diff
@@ -967,8 +995,8 @@ let sum_chain (type f) (module Circuit : Snark_intf.Run with type field = f)
 (* Definition of a gadget for a single foreign field addition
  *
  *    Inputs:
- *      full                  := Flag for whether to perform a full addition with valid_element check
- *                               on the result (default true) or just a single FFAdd row (false)
+ *      single                := Flag for whether to perform addition with valid_element check
+ *                               on the result (default false) or just a single FFAdd row (true)
```

Comment:
Because there's another flag later called full that made sense, so just to avoid confusion.

Reply:
But isn't the other `full` flag basically for the same concept? for adding the full set of checks?

Reply:
Well, it is indeed related. But I found that single was a bit more descriptive when you only wanted to do 1 addition, meaning to include a single row of FFAdd, not just one chain of 1 addition, that is ended with a Zero gate. To allow for further fine-grain access. But this could possibly change when we redefine how often we really need to check results, types of external checks, etc